src/resources/postcss/components/your-tickets/_attendees.pcss

```diff
@@ -0,0 +1,34 @@
+.tec-tickets__attendees-list-order-attendees {
+	.event-tickets-wallet-plus__your_tickets_title_area {
+		padding: 12px 0;
+	}
+
+	.tec-tickets__attendees-list-attendee-row {
+		align-items: center;
+		border-left: 1px solid var(--tec-color-border-secondary);
+		border-right: 1px solid var(--tec-color-border-secondary);
+		border-top: 1px solid var(--tec-color-border-secondary);
+		display: flex;
+		margin: 0;
+		padding: 20px;
+
+		&:last-child {
+			border-bottom: 1px solid var(--tec-color-border-secondary);
+		}
+	}
+
+	.tec-tickets__attendees-order-attendees-list-attendee-details {
+		align-self: center;
+		flex-grow: 1;
+	}
+
+	.event-tickets-wallet-plus__pdf-button {
+		align-self: center;
+		margin-left: auto;
+
+		a {
+			line-height: 14px;
+		}
+	}
```

Comment:
This modifier has to go in ETWP, looks like @codingmusician has already defined one [here](https://github.com/the-events-calendar/event-tickets-wallet-plus/blob/bucket/wallet-plus/src/resources/postcss/passes.pcss#L9)